### PR TITLE
Add test case for empty string input

### PR DESCRIPTION
**Pull request by OrdoFlammae**
Title says it all. I think the program should also check for an empty string, since that's another input that probably would be seen as invalid.

### CODE REVIEW

File: exercises/two-fer/example.lisp [hunk 1]
```diff
@@ -5,4 +5,4 @@
 (in-package #:two-fer)
 
 (defun twofer (name)
-  (format nil "One for ~a, one for me." (or name "you")))
+  (format nil "One for ~a, one for me." (if (or (null name) (equal name "")) "you" name)))
```
Comment (verdammelt) [inline exercises/two-fer/example.lisp]: We should use `string=` here. Generally the "rule" is to use the most specific equality operator possible.
